_data-prepper/log-analytics.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Log analytics
+nav_order: 12
+---
+
+# Introduction to log analytics
+
+Data Prepper is an extendable, configurable, and scalable solution for log ingestion into OpenSearch and Amazon OpenSearch Service. Data Prepper supports receiving logs from [Fluent Bit](https://fluentbit.io/) through the [HTTP Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md) and processing those logs with a [Grok Processor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/grok-processor/README.md) before ingesting them into OpenSearch through the [OpenSearch sink](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/opensearch/README.md).
```

Comment:
@carolxob , Just to give you more context for the restructure (#1542), some of these links should in the future be changed to the reference pages that will be created as part of #1542.

So for example,

`[HTTP Source](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-plugins/http-source/README.md)` - This would be a link to the new HTTP Source page which is part of the Data Prepper reference.

This should not hold this PR up, but I hope it gives you additional context!